<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>999</width>
    <height>747</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>589</width>
    <height>747</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton{
	background-color:white;
}
QWidget{
	background-color:white;
	border-style:outset;
}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="10">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_4" stretch="1,15">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="sizeConstraint">
        <enum>QLayout::SetMinAndMaxSize</enum>
       </property>
       <item>
        <widget class="QLabel" name="name_label">
         <property name="font">
          <font>
           <family>黑体</family>
           <pointsize>18</pointsize>
          </font>
         </property>
         <property name="text">
          <string>沙雕</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="remote_desktop_btn">
         <property name="minimumSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="baseSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton#remote_desktop_btn{
	background-color:white;
	
	background-image: url(:/noclick/noclick/远程桌面.png);
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="video_call_btn">
         <property name="minimumSize">
          <size>
           <width>40</width>
           <height>32</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton#video_call_btn{
	
	background-image: url(:/noclick/noclick/音视频交流.png);
	
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="group_chat">
         <property name="minimumSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="baseSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton#group_chat{
	
	background-image: url(:/noclick/noclick/创建群聊.png);
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="setting_btn">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton#setting_btn{
	
	background-image: url(:/noclick/noclick/设置.png);
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout" stretch="10,1,3">
       <item>
        <widget class="QWebEngineView" name="web_view">
         <property name="styleSheet">
          <string notr="true">QWebEnglineView#web_view{
	border-bottom-color:black;
	border-bottom-width:1px;
}</string>
         </property>
         <property name="url">
          <url>
           <string>about:blank</string>
          </url>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="font_btn">
           <property name="minimumSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton#font_btn{
	background-image: url(:/noclick/noclick/字体.png);
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="emoji_btn">
           <property name="minimumSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton#emoji_btn{
	background-image: url(:/noclick/noclick/笑脸.png);	
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="screenshot_btn">
           <property name="minimumSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton#screenshot_btn{
	
	background-image: url(:/noclick/noclick/剪切.png);
	
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="quake_btn">
           <property name="minimumSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton#quake_btn{
	
	background-image: url(:/noclick/noclick/抖动.png);
	
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="iconSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="voice_message_btn">
           <property name="minimumSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton#voice_message_btn{
	
	background-image: url(:/noclick/noclick/音频.png);
	
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="send_doc_btn">
           <property name="minimumSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton#send_doc_btn{
	
	background-image: url(:/noclick/noclick/文件夹.png);
	
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QComboBox" name="message_log_btn">
           <property name="minimumSize">
            <size>
             <width>90</width>
             <height>32</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>32</height>
            </size>
           </property>
           <item>
            <property name="text">
             <string>消息记录</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QWidget" name="widget" native="true">
           <widget class="QPushButton" name="send_btn">
            <property name="geometry">
             <rect>
              <x>870</x>
              <y>100</y>
              <width>93</width>
              <height>28</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton#send_btn{
	background-color:rgba(64, 141, 240, 90%);
	color:white;
	border-radius: 5px;
}
QPushButton#send_btn:hover{
	background-color:rgba(64, 141, 240, 100%);
	color:white;
	border-radius: 5px;
}
QPushButton#send_btn:pressed{
	background-color:rgba(64, 141, 259, 100%);
	color:white;
	border-radius: 5px;
}

</string>
            </property>
            <property name="text">
             <string>发送</string>
            </property>
            <property name="shortcut">
             <string>Return</string>
            </property>
           </widget>
           <widget class="QPlainTextEdit" name="text_edit">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>971</width>
              <height>141</height>
             </rect>
            </property>
           </widget>
           <zorder>text_edit</zorder>
           <zorder>send_btn</zorder>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebEngineView</class>
   <extends>QWidget</extends>
   <header location="global">QtWebEngineWidgets/QWebEngineView</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../image/icon.qrc"/>
 </resources>
 <connections/>
</ui>
